tests/diagnostics/snapshots/test_two_spans_different_lineno_lens.txt

```diff
@@ -0,0 +1,9 @@
+Error: Can't compare apples with oranges (at <unknown>:99:6)
+    | 
+ 97 | apple == orange
+ 98 | apple == orange
+ 99 | apple == orange
+    |       ^^ Comparison attempted here
+    | 
+100 | apple == orange
```

Comment:
This confused me a bit too - I realize this is purely formatting, but this looks like the subdiagnostic should *also* refer to line 99! (What happens if it does? Is that a separate test worth having?)

Changing the "This is an apple" to "This apple is on another line" might have avoided confusion ;)